sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -47,6 +46,7 @@ import {
   CoinMetadataStruct,
   GetObjectDataResponse,
   GetOwnedObjectsResponse,
+  GatewayTxSeqNumber,
```

Comment:
nit: lets revert all GatewayTxSeqNumber changes in this PR, we don't need to have this in commit history.